api/src/main/java/org/hawkular/inventory/api/MultipleEntityBrowser.java

```diff
@@ -23,7 +23,7 @@
  * @author Lukas Krejci
  * @since 1.0
  */
-interface MultipleEntityBrowser<Entity> {
+public interface MultipleEntityBrowser<Entity> {
```

Comment:
What about `ResolvableToMany` for `MultipleEntityBrowser` and `ResolvableToSingle` for `SingleEntityBrowser`?